scripts/components/CalendarListItem.jsx

```diff
@@ -0,0 +1,114 @@
+var React = require('react');
+var Radium = require('radium');
+var Moment = require('moment');
+var Api = require('../utils/Api.js');
+var User = require('../utils/User.js');
+var MuiContextified = require('./MuiContextified.jsx');
+var MaterialUI = require('material-ui');
+var MenuItem = require('material-ui/lib/menus/menu-item');
+var MenuDivider = require('material-ui/lib/menus/menu-divider');
+var IconMenu = MaterialUI.IconMenu;
+var ListItem = MaterialUI.ListItem;
+var ListDivider = MaterialUI.ListDivider;
+var IconButton = MaterialUI.IconButton;
+var Dialog = MaterialUI.Dialog;
+var FlatButton = MaterialUI.FlatButton;
+var Label = require('./Label.jsx');
+
+var CalendarListItem = React.createClass({
+  propTypes: {
+    event: React.PropTypes.object.isRequired,
+    refresh: React.PropTypes.func,
+  },
+
+  showEventDialog() {
+    this.refs.dialog.show();
+
+  },
+
+  deleteEvent() {
+    Api.deleteEvent({
+      eventId: this.props.event.id,
+      callback: (err, response) => {
+        if (err) {
+        console.log(err);
+        return reply(err);
```

Comment:
`reply()` is a Hapi.js serverside function.... Not sure why it is being used here.